src/org/opendatakit/briefcase/ui/reused/UI.java

```diff
@@ -64,6 +68,33 @@ public static JButton cellWithButton(JTable table, Object value, boolean isSelec
     return button;
   }
 
+  public static void infoMessage(String message) {
+    infoMessage(APP_NAME, message, false);
+  }
+
+  public static void infoMessage(String title, String message, boolean nonBlocking) {
+    if (nonBlocking) {
+      new SwingWorker() {
+        @Override
+        protected Object doInBackground() {
+          JOptionPane.showMessageDialog(buildDialogParent(), message, title, PLAIN_MESSAGE);
```

Comment:
Aren’t you violating this? https://docs.oracle.com/javase/7/docs/api/javax/swing/package-summary.html#threading